driver/driver.feature

```diff
@@ -0,0 +1,1265 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# These tests are dedicated to test the required functionality of TypeDB drivers. Can be used to test any client
+# application which aims to support all the operations presented in this file for the complete user experience.
+# The following steps are suitable for both CORE and CLOUD drivers. It is recommended to test both of them.
+# NOTE: for complete guarantees, all the drivers are also expected to cover the `connection` package.
+
+#noinspection CucumberUndefinedStep
+Feature: TypeDB Driver
+
+  Background: Open connection / create driver, create database
+    Given typedb starts
+    Given connection is open: false
+    Given connection opens with default authentication
+    Given connection is open: true
+    Given connection has 0 databases
+    Given connection create database: typedb
+    Given connection has database: typedb
+
+    # TODO: Add error strings checks
```

Comment:
Decided not to worry about it at the moment, as we don't check it anywhere in 3.0. But can start adding it in the driver's feature.